### PR TITLE
Upgrade Axios dependency

### PR DESCRIPTION
**Pull request by JuanVqz**
[Axios was released yesterday](https://github.com/axios/axios/releases/tag/v0.21.4)

Fixing JSON transform when data is stringified. Providing backward compatibility and complying to the JSON RFC standard (#4020)

Should I change the CHANGELOG?
I did it.

### CODE REVIEW

**Comment by akaraman85** [comment]
Ran into this issue with a security vulnerability scan.  👍 on the merge.

**Review by AJ1039593 — APPROVED** [review]

**Comment by raniesantos** [comment]
I also encountered this issue. Waiting for the merge.

**Comment by raniesantos** [comment]
Is defunctzombie the only one with permissions to merge this?

**Comment by TheBoroer** [comment]
@raniesantos yes he is.

@defunctzombie this look good to me to get merged in :) 
If he hapepns to not see this, I'll ping him in other channels

File: CHANGELOG.md
Comment (defunctzombie) [inline CHANGELOG.md:1]: PRs should not bump this. This is up to maintainer to decide what version bump to make.

**Review by defunctzombie — CHANGES_REQUESTED** [review]

**Review by defunctzombie — APPROVED** [review]

**Comment by raniesantos** [comment]
Thanks

**Comment by giamir** [comment]
@defunctzombie When a new version will be released in npm with this patch? Thanks

**Pull request merged**
